CMakeLists.txt

```diff
@@ -822,6 +822,17 @@ if(BUILD_TESTS)
     ADDITIONAL_ARGS -p liblogging --duration 45m
   )
 
+  add_e2e_sandbox_test(
```

Comment:
If you add an entry for this to `.cmake-format.py` it will be forwarded like our other CMake functions.

Reply:
Got it, it's forwarded now :)